frontend/common/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,145 @@
+import {
+  ArrowLeftIcon,
+  ArrowRightIcon,
+  DotsHorizontalIcon,
+} from "@heroicons/react/solid";
+import * as React from "react";
+import { useIntl } from "react-intl";
+import { Button } from "..";
+import { DOTS, usePagination } from "./usePagination";
+
+export interface PaginationProps {
+  /** The current active page. */
+  currentPage: number;
+  /** Total count of items from array. */
+  totalCount: number;
+  /** The minimum number of page buttons to be shown on each side of the current page button. */
+  siblingCount: number;
+  /** The maximum number of visible items on a single page. */
+  pageSize: number;
+  /** Callback that changes to the page number value. */
+  handlePageChange: (pageNumber: number) => void;
+}
+
+const Pagination: React.FunctionComponent<PaginationProps> = ({
+  totalCount,
+  siblingCount,
+  currentPage,
+  pageSize,
+  handlePageChange,
+}) => {
+  const intl = useIntl();
+  const paginationRange = usePagination({
+    currentPage,
+    totalCount,
+    siblingCount,
+    pageSize,
+  });
+
+  // If there are less than 2 times in pagination range we shall not render the component
+  if (currentPage === 0 || paginationRange.length < 2) {
+    return null;
+  }
+
+  const nextPage = () => {
+    handlePageChange(currentPage + 1);
+  };
+
+  const previousPage = () => {
+    handlePageChange(currentPage - 1);
+  };
+
+  const lastPage = paginationRange[paginationRange.length - 1];
+  return (
+    <nav
+      role="navigation"
+      aria-label={intl.formatMessage({
+        defaultMessage: "Pagination Navigation",
+        description: "Aria label for pagination nav",
+      })}
+    >
+      <ul
+        className="reset-ul"
```

Comment:
Ah hey, learned a thing! https://stackoverflow.com/a/19890265

Seems this class needs to also use `list-style-type: none` in addition to `list-style: none`

Reply:
Yea that's right! However, I'm confused as to why I wasn't seeing the bullets on my end 😕

Reply:
Maybe an artifact of merging storybooks that CI does? Were you looking at a normal "common" or "admin" storybook?

It might be that when merged, something else (person or package) sets a more specific `list-style-type` CSS tag somewhere, and so `list-type` isn't priority? (Not sure... lots about CSS is a bit magic to me 😬 )

Reply:
I think the common.css file isn't getting added into the storybook build in Chromatic. I'm going to make an issue for it.